rust-connectors/sinks/sql/README.md

```diff
@@ -0,0 +1,98 @@
+# Fluvio SQL Sink connector
+The SQL Sink connector reads records from Fluvio topic, applies configured transformations, and 
+sends new records to the SQL database.
```

Comment:
I think that we should be explicit and to say that this only executes `INSERT` commands

Reply:
Done